runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/DataflowMetricsTest.java

```diff
@@ -184,6 +189,13 @@ private MetricUpdate makeCounterMetricUpdate(
     return setStructuredName(update, name, namespace, step, tentative);
   }
 
+  private MetricUpdate makeStringSetMetricUpdate(
+      String name, String namespace, String step, List<String> setValues, boolean tentative) {
```

Comment:
Why is this a List<String> of values? Should it be a Set<String> or Container<String>?